templates/posts/detail_anonymous.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+{% load static %}
+
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="{% static 'css/global.css'%}" type="text/css">
+    <meta charset="UTF-8">
+    <title>Recipes</title>
+</head>
+<body>
+<div style="margin: 0 auto;width: 500px">
+    {% if request.user.is_authenticated %}
+    <h2><a href="{% url 'posts:post_list' %}"> RECIPES </a></h2>
+    <h2><a href="{% url 'posts:post_create' %}"> SHARE </a></h2>
+    <h2><a href="{% url 'users:logout' %}"> LOGOUT </a></h2>
+    {% else %}
+    <h2><a href="{% url 'posts:post_list' %}"> RECIPES </a></h2>
+    <h2><a href="{% url 'users:register' %}"> REGISTER </a></h2>
+    <h2><a href="{% url 'users:login' %}"> LOGIN </a></h2>
+    {% endif %}
+</div>
+<hr>
+<h2>{{ post.title }}</h2>
+<p> by {{ post.user.first_name }} {{ post.user.last_name }} - {{ post.created_date }} </p>
+<img src="{{ post.img.url }}">
```

Comment:
Please do not use abbreviation for field names and rename `img` to `image`.
Also rename `created_date` to `date_created` and `updated_date` to `date_updated`